src/main/java/com/paskar/email/application/configuration/SpringConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.paskar.email.application.configuration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import org.springframework.boot.autoconfigure.jdbc.JdbcTemplateAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+
+@Configuration(proxyBeanMethods = false)
+public class SpringConfig {
+    @Bean
+    public ObjectMapper mapper() {
+        return new Jackson2ObjectMapperBuilder()
+                .indentOutput(true)
+                .propertyNamingStrategy(PropertyNamingStrategy.UPPER_CAMEL_CASE)
+                .build();
+    }
+
+    @Bean
+    public JdbcTemplateAutoConfiguration getJdbcTemplate() {
```

Comment:
AutoConfiguration class is a class that does some logic instead of you (Spring Boot).
You should not create these classes.